src/oscar/apps/basket/abstract_models.py

```diff
@@ -736,7 +736,7 @@ def get_price_breakdown(self):
             item_incl_tax_discount = (
                 self.discount_value / int(self.consumer.consumed()))
             item_excl_tax_discount = item_incl_tax_discount * self._tax_ratio
-            item_excl_tax_discount = item_excl_tax_discount.quantize(D('0.01'))
+            item_excl_tax_discount = round_half_up(item_excl_tax_discount)
```

Comment:
Just looking at the historic way in which stuff has been rounded, I think we should probably round *down* here and everywhere else that we are working out a discount. That is the more conservative approach (i.e., you will never end up giving a higher discount that you intended to).

Reply:
I will have to do some more testing to check that out. Do you mean really round down eg. 1.499 = 1.49 or do you mean round half down eg. 1.499 = 1.50 but 1.495 = 1.49 ?

Reply:
Just realised that is this setting for offer rounding - we should probably be using that? https://github.com/django-oscar/django-oscar/blob/f2bb0fb73c83cb879fb2dec9184059dd96d814e0/docs/source/ref/settings.rst#oscar_offer_rounding_function

Reply:
I looked into that before and the only place it is used is in AbstractBenefit. Note that I didn't really change anything, I only fixed usage of the really weird and erroneous behaviour of the default rounding function. It has to be investigated if this is the right place to apply the round down. More testing would be needed.

Reply:
Spent a bit more time looking at this, and I think what you have is fine.

Reply:
The value being rounded is the item_excl_tax_discount, which is the item price. If I round it down there is actually MORE discount given in most cases. I don't think that is great idea.